scripts/check_layer_size.sh

```diff
@@ -14,26 +14,22 @@ MAX_LAYER_UNCOMPRESSED_SIZE_KB=$(expr 24 \* 1024)
 
 LAYER_FILES_PREFIX="datadog_lambda_py"
 LAYER_DIR=".layers"
```

Comment:
This check now only lives in gitlab and doesn't need to be ran once for all versions.